packages/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.3
+
+* Massive refactoring of Android Camera functionality to allow reuse of Camera class.
```

Comment:
I don't believe refactors require a version bump if no new features were added. If someone accessing this from `pub.dev` they would notice a difference on the dart side.

Reply:
So we don't publish this at all? That's fine by me. Reverting.